tyk-oss/README.md

```diff
@@ -230,19 +230,21 @@ If you want to use your own key/cert pair, you must follow the following steps:
 An Ingress resource is created if `tyk-gateway.gateway.ingress.enabled` is set to true.
 
 ```yaml
+    # Creates an ingress object in k8s. Will require an ingress-controller and
+    # annotation to that ingress controller.
     ingress:
       # if enabled, creates an ingress resource for the gateway
-      enabled: true
+      enabled: false
```

Comment:
Shouldn't it be set to true since we are providing an example?

Reply:
yes, it must be true

Reply:
updated accordingly in https://github.com/TykTechnologies/tyk-charts/pull/191/commits/c2e0cfc0ad304dfd75751be442378f9eb16f3af9